modules/zone/zone.go

```diff
@@ -189,52 +156,44 @@ func (s *RoutineLookupFactory) MakeLookup() (zdns.Lookup, error) {
 //
 type GlobalLookupFactory struct {
 	zdns.BaseGlobalLookupFactory
-	Glue        *map[string][]string
-	GlueLock    sync.RWMutex
-	Subfactory  alookup.GlobalLookupFactory
-	CacheFactor int
-	CacheHash   *cachehash.CacheHash
-	CHmu        sync.Mutex
-	Manager     CallbackManager
+	Glue         *map[string][]string
+	GlueLock     sync.Mutex
+	Subfactory   alookup.GlobalLookupFactory
+	CacheFactor  int
+	GlueFilePath string
+	Hash         *cachehash.CacheHash
+	GlueCache    *cachehash.CacheHash
+	CHmu         sync.Mutex
 }
 
 func (s *GlobalLookupFactory) AddFlags(f *flag.FlagSet) {
 	f.BoolVar(&s.Subfactory.IPv4Lookup, "ipv4-lookup", true, "perform A lookups for each server")
 	f.BoolVar(&s.Subfactory.IPv6Lookup, "ipv6-lookup", true, "perform AAAA record lookups for each server")
-	f.IntVar(&s.CacheFactor, "cache-size-factor", 25, "number of times larger than the number of threads to make the cache of successes")
+	f.IntVar(&s.CacheFactor, "cache-size-factor", 25, "number of times larger than the number of threads to make the cache for looked up nameservers")
+	f.StringVar(&s.GlueFilePath, "glue-file", "", "glue file path")
```

Comment:
I thought we were doing two passes over the original file instead of requiring it to be passed in twice. What happened to that approach?